package.json

```diff
@@ -5,7 +5,7 @@
   "main": "dist/index.cjs.js",
   "module": "dist/index.esm.js",
   "files": [
-    "src",
+    "src/*.js",
```

Comment:
This will include test files, but why do we include `src` anyway? I think just `dist`, right?

Reply:
This will not include source files. `src/**/*.js` will. I publish `src` because of flow types. I prefer to reuse instead of copying.

Reply:
We can simply move tests folder to the root.